src/pages/en.md

```diff
@@ -316,6 +316,10 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
+- [ ] Promptly update version of gem at RubyGems
```

Comment:
I think the word "promptly" is too strong. Not every fix/feature should cause an immediate release.

We should provide some clarification, for example:

- security fix – release it!

- fixing a regression bug – release it! (Example: [Ruby 2.3.3](https://www.ruby-lang.org/en/news/2016/11/21/ruby-2-3-3-released/))

- adding a feature – wait for next planned release

- fixing a bug that was there for a long time – wait for next planned release.

Also, I'd like to add a note about RC versions, e.g. "Publish RC/beta version prior to releasing a major update".